src/lib.rs

```diff
@@ -147,6 +145,28 @@ pub trait Rng: RngCore {
     /// println!("{}", x);
     /// println!("{:?}", rng.gen::<(f64, bool)>());
     /// ```
+    ///
+    /// # Arrays and tuples
+    ///
+    /// The `rng.gen()` method is able to generate arrays (up to 32 elements)
+    /// and tuples (up to 12 elements), so long as all element types can be
+    /// generated.
+    ///
+    /// For arrays of integers, especially for those with small element types
+    /// (< 64 bit), it will likely be faster to instead use [`Rng::fill`].
```

Comment:
I think we can make `gen()` use `Rng::fill` with some breaking changes.

Reply:
Nevermind, this is discussed in #269. I agree it is better to suggest using `fill` if performance is important.